### ISSUE TITLE
Session TTLs With Redis

### ISSUE THREAD

**Issue by cdx-grillo** [magro/memcached-session-manager#356]
Thanks for the awesome product!  Quick question, my session is set to expire in the web.xml as <session-timeout>240</session-timeout> (4 hours).   When I look at a sample session key TTL in redis, I see something like:
>TTL "85224AE603537F71E315227034067D18"
(integer) 21965
so 6 hours in Redis.  I saw a previous post where the web.xml session timeout was not being honored but sounded like that was fixed with a 1.9.X version and I'm using 2.1.1.  

Any know issues with the redis TTL not in sync with the session timeout defined in the web.xml? 

Thanks much,

Greg

**Comment by magro** [comment]
Is the session timeout / maxInactiveInterval defined in server.xml/context.xml? This one would be preferred over web.xml.

**Comment by cdx-grillo** [comment]
No, was just the web.xml, thanks very much, I'll take a look at defining in the server or context.xml
